checks/image/image_check.go

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright 2019 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/digitalocean/clusterlint/checks"
+	"github.com/digitalocean/clusterlint/kube"
+	"github.com/docker/distribution/reference"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func init() {
+	checks.Register(&containerImageCheck{})
+}
+
+type containerImageCheck struct{}
+
+// Name returns a unique name for this check.
+func (pc *containerImageCheck) Name() string {
+	return "container-images"
+}
+
+// Groups returns a list of group names this check should be part of.
+func (pc *containerImageCheck) Groups() []string {
+	return []string{"image"}
```

Comment:
@adamwg - I like adding checks based on what object the check deals with. Maybe we should do that for all our checks and then allow users to run say just pod related checks or, node related checks.. WDYT? @nathanjsweet - how about changing this to pod then?

Reply:
Yeah, that's a good idea.